documents/src/pages/elements/select.md

```diff
@@ -380,3 +380,38 @@ ef-select {
 | ------------------ | ------------------------------------ |
 | --list-max-height  | Maximum height of the drop-down list |
 | --list-max-width   | Maximum width of the drop-down list  |
+
+## Accessibility
+::a11y-intro::
+
+`ef-select` is assigned `role="listbox"` and can include properties such as `aria-expanded` and `aria-activedescendant`. Select options are assigned `role="option"` and can include properties such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  
```

Comment:
```suggestion
`ef-select` is assigned `role="listbox"` and can include attributes such as `aria-expanded` and `aria-activedescendant`. Select options are assigned `role="option"` and can include attributes such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  
```

Reply:
`ef-select` is actually assigned `role="button"` and does not use `aria-activedescendant` instead it focuses on item directly. Also thin